cypress/support/commands.ts

```diff
@@ -575,16 +575,27 @@ export const createToken = (
 
 // TODO: have to go through setup because we cannot create a user w/ a password via the user API
 export const setupUser = (): Cypress.Chainable<Cypress.Response<any>> => {
-  return cy.request({
-    method: 'GET',
-    url: '/debug/provision',
-  })
+  const defaultUser = Cypress.env('defaultUser')
+  const userParam = defaultUser ? `?user=${defaultUser}` : ''
```

Comment:
the first time, defaultUser will be empty. Every time after that, it will be populated